crates/cli-flags/src/lib.rs

```diff
@@ -211,6 +211,9 @@ wasmtime_option_group! {
         pub timeout: Option<Duration>,
         /// Size of stacks created with cont.new instructions
         pub wasmfx_stack_size: Option<usize>,
+        /// Space that must be left on stack when starting execution of a
+        /// function while running on a continuation stack.
+        pub wasmfx_red_zone_size: Option<usize>,
```

Comment:
presumably this is platform/abi dependent? Like SysV mandates 16 bytes alignment. I suppose we round up when actually allocate the stack. Nonetheless, it may be worth writing here that the size given here is an underapproximation.

Reply:
As discussed elsewhere, this value does not influence the allocation of stacks; it isn't really constrained at all in that regard. The only condition is that it should be smaller than the stack size. I've added a comment stating this condition, even though we don't currently check it.